napari/layers/shapes/_accelerated_triangulate.py

```diff
@@ -0,0 +1,510 @@
+from __future__ import annotations
```

Comment:
Adding a 1 line module docstring at top of file would be helpful for future contributors/maintainers.
Perhaps:

```suggestion
"""Triangulation utilities"""
from __future__ import annotations
```